Makefile

```diff
@@ -982,52 +985,17 @@ install: build
 	cp -f $(BUILDDIR)/teleport  $(BINDIR)/
 	mkdir -p $(DATADIR)
 
-
 # Docker image build. Always build the binaries themselves within docker (see
 # the "docker" rule) to avoid dependencies on the host libc version.
 .PHONY: image
-image: clean docker-binaries
+image: OS=linux
+image: TARBALL_PATH_SECTION:=-s "$(shell pwd)"
+image: clean docker-binaries build-archive oss-deb
```

Comment:
I assume that running this locally is suppose to build the deb package from the outputs of build-archive. Currently build-common.sh attempts to pull the image version shown in the Makefile and not use your locally built build-archive. 

Additionally this step is also not working correctly due to issues in the changes to the e reference. I would maybe make sure the e commit you are referencing is up to date.

Reply:
Setting "build-archive" as a prerequisite to build a tarball and setting `TARBALL_PATH_SECTION:=-s "$(shell pwd)"` will result in `build-common.sh` using the locally built tarball rather than pulling from `https://get.gravitational.com/`